src/css/themes/po-theme-default.css

```
@@ -1570,9 +1570,18 @@ po-textarea {
   /*------------------------------------*\
     TOOLTIP
   \*------------------------------------*/
+}
 
-  --color-tooltip-background-color-balloon: var(--color-neutral);
-  --color-tooltip-color-text-balloon: var(--color-white);
+.po-tooltip {
+  --color: var(--color-neutral-dark-40);
+  --border-radius: var(--border-radius-md);
+  --font-family: var(--font-family-theme);
+  --text-color: var(--color-white);
```

Alterar porque a var `--color-white` vai ser depreciada.

```suggestion
  --text-color: var(--color-neutral-light-00);
```

---

o `color-white` recebe a variável `--color-neutral-light-00`, essa forma é para manter a customização atual.